caffe2/operators/upsample_op.cu

```diff
@@ -1,3 +1,5 @@
+// copied and pasted from pytorch to test if this passes the build.
```

Comment:
Just checking -- you are modifying Caffe2's bilinear upsampling code?

Reply:
You can ignore that comment. that was because I was failing some tests but when I checked locally against the unmodified upsampling code, the results were identical. So I was wondering if the tests were faulty or the original code had errors. Turns out I wasn't iterating over the batch dimension.

Yes, I am modifying Caffe2's bilinear upsampling code.